### PR TITLE
Copy URL (files, shares, blobs, containers)

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
Fixes #116

File: src/azureStorageExplorer/fileShares/fileShareNode.ts [hunk 1]
```diff
@@ -50,6 +54,14 @@ export class FileShareNode implements IAzureParentTreeItem {
             }));
     }
 
+    public async copyUrl(_node: IAzureParentNode): Promise<void> {
+        let fileService = azureStorage.createFileService(this.storageAccount.name, this.key.value);
+        let url = fileService.getUrl(this.share.name, "");
+        copypaste.copy(url);
+        azureStorageOutputChannel.show();
+        azureStorageOutputChannel.appendLine(`Share URL copied to clipboard: ${url}`);
+    }
```
Comment (PrashanthCorp) [inline src/azureStorageExplorer/fileShares/fileShareNode.ts]: I notice that in all the use cases, the only difference is line 1, in the method being called. 
Is there an easy way to remove the duplicated code (getUrl to appendLine)? 

I think having a createService method (with `FileShareNode` calling `createFileService`, etc.) would free up the other lines to a common method call.

Comment (StephenWeatherford) [reply src/azureStorageExplorer/fileShares/fileShareNode.ts]: I considered it but decided it wouldn't be worth the gain, given that the logic is dead simple.

Comment (StephenWeatherford) [reply src/azureStorageExplorer/fileShares/fileShareNode.ts]: ping

**Pull request merged**